libs/@hashintel/design-system/src/autocomplete-dropdown.tsx

```diff
@@ -37,50 +38,53 @@ export const AutocompleteDropdown = ({
       />
       <Paper
         {...props}
-        sx={(theme) => ({
-          p: 1,
-          border: 1,
-          boxSizing: "border-box",
-          borderColor: theme.palette.gray[30],
-          boxShadow: "none",
-          [`${popperPlacementSelectors.top} &`]: {
-            borderBottom: 0,
-            borderBottomLeftRadius: 0,
-            borderBottomRightRadius: 0,
-          },
-          [`${popperPlacementSelectors.topStart} &`]: {
-            borderBottom: 0,
-            borderBottomLeftRadius: 0,
-          },
-          [`${popperPlacementSelectors.bottom} &`]: {
-            borderTop: 0,
-            borderTopLeftRadius: 0,
-            borderTopRightRadius: 0,
-          },
-          [`${popperPlacementSelectors.bottomStart} &`]: {
-            borderTop: 0,
-            borderTopLeftRadius: 0,
-          },
-
-          [`.${autocompleteClasses.listbox}`]: { p: 0 },
-          [`.${autocompleteClasses.noOptions}`]: { display: "none" },
-          [`.${autocompleteClasses.option}`]: {
-            borderRadius: 1,
-            my: 0.25,
-
-            [`&[aria-selected="true"]`]: {
-              backgroundColor: `${theme.palette.blue[20]} !important`,
+        sx={[
+          (theme) => ({
+            p: 1,
+            border: 1,
+            boxSizing: "border-box",
+            borderColor: theme.palette.gray[30],
+            boxShadow: "none",
+            [`${popperPlacementSelectors.top} &`]: {
+              borderBottom: 0,
+              borderBottomLeftRadius: 0,
+              borderBottomRightRadius: 0,
+            },
+            [`${popperPlacementSelectors.topStart} &`]: {
+              borderBottom: 0,
+              borderBottomLeftRadius: 0,
+            },
+            [`${popperPlacementSelectors.bottom} &`]: {
+              borderTop: 0,
+              borderTopLeftRadius: 0,
+              borderTopRightRadius: 0,
+            },
+            [`${popperPlacementSelectors.bottomStart} &`]: {
+              borderTop: 0,
+              borderTopLeftRadius: 0,
             },
 
-            "&.Mui-focused": {
-              backgroundColor: `${theme.palette.gray[10]} !important`,
+            [`.${autocompleteClasses.listbox}`]: { p: 0 },
+            [`.${autocompleteClasses.noOptions}`]: { display: "none" },
+            [`.${autocompleteClasses.option}`]: {
+              borderRadius: 1,
+              my: 0.25,
 
               [`&[aria-selected="true"]`]: {
-                backgroundColor: `${theme.palette.gray[20]} !important`,
+                backgroundColor: `${theme.palette.blue[20]} !important`,
+              },
+
+              "&.Mui-focused": {
+                backgroundColor: `${theme.palette.gray[10]} !important`,
+
+                [`&[aria-selected="true"]`]: {
+                  backgroundColor: `${theme.palette.gray[20]} !important`,
+                },
               },
```

Comment:
Similar comment, why the need for using `!important`?

Reply:
Same as above